presentation/src/main/java/com/stop/ui/map/MapFragment.kt

```diff
@@ -0,0 +1,133 @@
+package com.stop.ui.map
+
+import android.Manifest.permission
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import androidx.core.content.ContextCompat.checkSelfPermission
+import androidx.core.graphics.drawable.toBitmap
+import androidx.fragment.app.Fragment
+import com.skt.tmap.TMapGpsManager
+import com.skt.tmap.TMapView
+import com.skt.tmap.overlay.TMapMarkerItem
+import com.stop.R
+import com.stop.databinding.FragmentMapBinding
+
+class MapFragment : Fragment() {
+    private var _binding: FragmentMapBinding? = null
+    private val binding get() = _binding!!
+
+    private lateinit var tMapView: TMapView
+    private var isTracking = true
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentMapBinding.inflate(inflater, container, false)
+        initBinding()
+
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initView()
+        initTMap()
+    }
+
+    private fun initBinding() {
+        // TODO lifecycleOwner, viewModel 설정
+    }
+
+    private fun initView() {
+        binding.imageViewCompassMode.setOnClickListener {
+            tMapView.isCompassMode = tMapView.isCompassMode.not()
+        }
+
+        binding.imageViewCurrentLocation.setOnClickListener {
+            tMapView.setCenterPoint(tMapView.locationPoint.latitude, tMapView.locationPoint.longitude, true)
+            isTracking = true
+        }
+
+        binding.imageViewBookmark.setOnClickListener {
+
+        }
+    }
+
+    private fun initTMap() {
+        tMapView = TMapView(requireContext())
+        tMapView.setSKTMapApiKey(T_MAP_API_KEY)
+        tMapView.setOnMapReadyListener {
+            tMapView.mapType = TMapView.MapType.NIGHT
+            tMapView.zoomLevel = 16
+            setTrackingMode()
+        }
+        tMapView.setOnEnableScrollWithZoomLevelListener { _, _ ->
+            isTracking = false
+        }
+
+        binding.frameLayoutContainer.addView(tMapView)
+    }
+
+    private fun setTrackingMode() {
+        if (isLocationPermissionsGranted()) {
+            val manager = TMapGpsManager(requireContext()).apply {
+                minDistance = 2.5F
+                provider = TMapGpsManager.PROVIDER_GPS
+                openGps()
+                provider = TMapGpsManager.PROVIDER_NETWORK
+                openGps()
+            }
+
+            manager.setOnLocationChangeListener(onLocationChangeListener)
+        }
+    }
```

Comment:
```suggestion
    private fun setTrackingMode() {
        if (isLocationPermissionsGranted().not()) {
            return
        }
        val manager = TMapGpsManager(requireContext()).apply {
            this.minDistance = 2.5F
            this.provider = TMapGpsManager.PROVIDER_GPS
            this.openGps()
            this.provider = TMapGpsManager.PROVIDER_NETWORK
            this.openGps()
        }

        manager.setOnLocationChangeListener(onLocationChangeListener)
    }
```
if문이 true일 때만 동작하는 경우, 위와 같이 수정하면 depth를 줄일 수 있어서 가독성이 좋아져요.